thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/datalayer/dto/OnlineDetectionDataDTO.java

```diff
@@ -1,6 +0,0 @@
-package org.apache.pinot.thirdeye.datalayer.dto;
```

Comment:
Can you move these classes instead of deleting and recreating a new one?

Reply:
This is all done by Intellij. Some of these classes also had code changes which resulted in it showing up as create/delete instead of move. But as such all the classes were git "moved".